output/cloud/expv2/pbcloud/metric.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
+// 	protoc-gen-go v1.30.0
```

Comment:
I don't think we need this change here

Reply:
Done in [a95ba44](https://github.com/grafana/k6/pull/3100/commits/a95ba44757fc39ec406624dd8eb1ade7c1c43dff).